laser_encoders/language_list.py

```diff
@@ -0,0 +1,65 @@
+# Language mapping to handle different language codes and names
+LASER3_LANGUAGE = {
```

Comment:
You can start by perhaps using this list here:

https://github.com/facebookresearch/LASER/tree/main/nllb#list-of-available-laser3-encoders

If has a mapping for all available LASER3 encoders with the language name and language code.

Reply:
Done 😊